src/aliases.json

```diff
@@ -0,0 +1 @@
 "PapiWallace", "MEG": "Shocky", "rules": "Shocky", "CraNky_SnaiL": "CranKy_SnaiL", "RemosMutter.": "RemosMutter", "gradient": "Gradient", "GG~": "Iw3T.DarK", "DowaQ": "DowaQ", "NightCrawler.PE": "Crawler", "spg]love": "Love", "tS.Rob": "TS.Rob", "IamTry": "DreAm", "120": "MrHuDat", "minime": "Acx", "LinoBSC": "LinoBSC", "_pb": "NanaSan", "InFs]LaacTiger": "LaacTiger", "jugandoEbrio": "Blazz", "FatRedBelly": "FatRedBelly", "BlOoDy": "BlOoDy", "niGhT-BlOoDy": "BlOoDy", "Cholipandro123": "Chobo%20%28Peruvian%20player%29", "EXOLAG": "Iblazatar", "Aw]Swift": "Swift", "Stucell": "Stucell", "^Divine": "Divine", "DF_Divine": "Divine", "Divine^": "Divine", "Flame[Skynet].": "Flame", "Edgar018": "Edgar04", "InFs]Edgar018": "Edgar04", "Yami_": "Aeb", "sPG]aeb": "Aeb", "spg.aeb": "Aeb", "Ceron": "Ceron", "MasterV64": "Ghost%20OL", "kiuda": "Kiuda", "Aw]BlackWarwolf": "Blackwarwolf", "InFs]Psycho": "RusoopE", "InFs]elves": "Gnc", "InFs)DeSiRe": "DeS1Re", "TryingNewBuilds": "KiWiKaKi", "xcarlx": "Xcarlx", "rott": "Rott", "Madnesss": "Satiini", "Satiini.fi": "Satiini", "rp.seifer": "Rp.seifer", "papaChiko": "Chiko", "wakadaka": "HoT", "flow": "Flow", "CoNsTriVe": "PLZLEAVEDUCK", "deagledonny": "Donny", "kuPsen": "Donny", "RAGE-EdGE": "Edge", "LaG.EdGE": "Edge", "girls-love-me": "Revolta", "Che": "ReseT", "che": "ReseT", "sPG]Thanatos": "Thanatos", "Pasodoble": "Mosura", "zbychu": "Zbychu", "fantasteec": "LucifroN", "MrThreepWood": "VortiX", "Barcode": "VortiX", "InFs]benjita": "Benjita", "QuePasoAqui": "QuePasoAqui", "Alf": "Alf", "AJJA": "AJJA", "doj.": "Doj", "FiR": "FiR", "InFs]0Cervantes": "0Cervantes", "STLER": "STLER", "HartCrane": "HartCrane", "wut_": "AcidRain", "WellplayedTV": "WellPlayed", "PaaWciO": "PaaWciO", "RainbowDash": "Lucian", "Moonwells": "Moonwells", "TeAmoLiesel": "Madoka%20%28Peruvian%20player%29", "Berbatov": "Sixb0nes", "hellmoot": "Hellmoot", "CalmWaves": "CrunCher", "MagicO": "MagicO", "lort_rafau": "Lort_rafau", "Hunter_II": "Hunter_II", "Aruna": "Aruna", "hocuspocus": "Hocuspocus", "floss2xdaily": "Floss2xdaily", "buffMyKotg": "SoulEater", "ShilOhS": "ShilOhS", "Espejo": "Espejo", "laaaaaggggg11": "AleX1tO", "HonorSpren": "NoeXxX", "DaRk_DrAg0n": "DaRk%20DrAg0n", "WarBoxe": "WarBoxe", "UberUndead": "PhilBoT", "KeeperCopyright": "Sevazhu", "181027": "Moon%20%28Bulgarian%20player%29", "eX.GeNDo": "EX.GeNDo", "aarnikratti": "Aarnikratti", "Robinson": "Robinson", "Rigeteey": "Rigety", "GraVesc2": "GraVe", "wanderbraun": "Wanderbraun", "aMMo": "AMMMo", "Baruto": "Baruto", "ChestnutBarrow": "DArk", "TBW)Wopyy": "Wopy", "AretaS": "Aretas", "d3r_schosch": "D3r_schosch", "RealdogXL": "Shear", "Gatzu-": "Gatzu", "Humpty_Hump": "Humpty%20hump", "aLysoN9GoFF": "AWalk", "OveR.Eierhals": "DaNii", "RAGE.Believe": "Believe%20%28Canadian%20player%29", "Mantalo": "Mantalo", "Slovakiaguy": "SlovakiaGuy", "StoPClick": "StoPClick", "JoZen": "RaGman", "Stillownya": "Stillownya", "Mahoma": "Mahoma", "TheRearAdmiral": "TheRearAdmiral", "Zanjan": "Zanjan"}
```

Comment:
Looks like there are some issues if there are spaces or `(`, `)` characters in the aka. See Rage.Believe for example